ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot

```diff
@@ -0,0 +1,127 @@
+*** Settings ***
+Documentation     Codeflare operator E2E tests - https://github.com/opendatahub-io/codeflare-operator/tree/main/test/odh
+Suite Setup       Prepare Codeflare E2E Test Suite
+Suite Teardown    Teardown Codeflare E2E Test Suite
+Library           OperatingSystem
+Library           Process
+
+
+*** Variables ***
+${CODEFLARE_DIR}                codeflare-operator
+${CODEFLARE_REPO_URL}           %{CODEFLARE_REPO_URL=https://github.com/opendatahub-io/codeflare-operator.git}
+${CODEFLARE_REPO_BRANCH}        %{CODEFLARE_REPO_BRANCH=main}
+${ODH_NAMESPACE}                %{ODH_NAMESPACE=redhat-ods-applications}
+${NOTEBOOK_IMAGE_STREAM_NAME}   %{NOTEBOOK_IMAGE_STREAM_NAME=s2i-generic-data-science-notebook}
+
+
+*** Test Cases ***
+Run TestMNISTPyTorchMCAD E2E test
```

Comment:
@jiripetrlik How long it will take to run all these tests . Depending on that we can classify them whether to be placed in Sanity/Tier1 or in Tier2.

Reply:
It depends on available resources and network connection.
I assume it may take 5-10 minutes for every test.